WasatchNET/EEPROM.cs

```diff
@@ -1951,7 +1951,7 @@ void enforceReasonableDefaults()
                 wavecalCoeffs[4] = 0;
             }
 
-            if (minIntegrationTimeMS < 1)
+            if (minIntegrationTimeMS < 1 || minIntegrationTimeMS == 65535)

```

Comment:
Honestly, I should probably make that >5000.  No WP spectrometer is going to have a minimum integration more than 5sec (or even 1sec).